### PR TITLE
Charset categories

### PR DESCRIPTION
**Pull request by RoelN**

### CODE REVIEW

File: fontData.js [hunk 1]
```diff
@@ -12,7 +13,107 @@ const buildAxes = font => {
 };
 
 const buildChars = font => {
```
Comment (pascalw) [inline fontData.js:15]: Can you update the test cases for this function @RoelN?

Comment (pascalw) [reply fontData.js:15]: The output of this function now looks like this. Is that intentional? If yes, can you document what it means?

```json
{"Letter": {"Lowercase": {"latin": ["0061", "0062", "0063", "0064", "0065", "0066", "0067", "0068", "0069", "006A", "006B", "006C", "006D", "006E", "006F", "0070", "0071", "0072", "0073", "0074", "0075", "0076", "0077", "0078", "0079", "007A"]}, "Uppercase": {"latin": ["0041", "0042", "0043", "0044", "0045", "0046", "0047", "0048", "0049", "004A", "004B", "004C", "004D", "004E", "004F", "0050", "0051", "0052", "0053", "0054", "0055", "0056", "0057", "0058", "0059", "005A"]}}, "Number": {"Decimal Digit": {"undefined": ["0030", "0031", "0032", "0033", "0034", "0035", "0036", "0037", "0038", "0039"]}}, "Punctuation": {"Dash": {"undefined": ["002D", "005F", "2013", "2014"]}, "Parenthesis": {"undefined": ["0028", "0029", "005B", "005D", "007B", "007D"]}, "Quote": {"undefined": ["0022", "0027", "00AB", "00BB", "2018", "2019", "201C", "201D", "2039", "203A"]}, "undefined": {"undefined": ["0021", "002C", "002E", "002F", "003A", "003B", "003F", "005C", "00A1", "00BF", "2026"]}}, "Separator": {"Space": {"undefined": ["0020", "00A0"]}}, "Symbol": {"undefined": {"undefined": ["0026", "007C"]}}}```

Comment (RoelN) [reply fontData.js:15]: As discussed IRL, this is indeed what we need for now. But this structure could be improved so I'll give that a shot.

**Comment by RoelN** [comment]
As discussed offline, let's change the data structure to something easier:

```

  [
    {
      category: "Letter",
      subCategory: "Lowercase",
      script: "latin",
      chars: [
      ]
    },
    {
      category: "Punctuation",
      subCategory: null,
      script: null,
      chars: [
      ]
    }
  ]
```

This'll greatly simplify looping over all character categories.

**Review by pascalw — APPROVED** [review]
As discussed I'm approving blindly due to lack of time and you needing to make a deadline. I'll have a look later.

**Pull request merged**